src/OpenLoco/OpenLoco.cpp

```diff
@@ -218,7 +218,12 @@ namespace OpenLoco
     void setGameSpeed(uint8_t speed)
     {
         assert(speed <= 3);
```

Comment:
You will need to check for this within the game command and return FAILURE as we can't have a game command that causes an assert to trigger.

Reply:
Will do. Is there a place to put enums/constants, as I'd like to make the speed value an enum but I'm not sure where to put it.